src/components/LeafletMap.js

```diff
@@ -250,4 +253,24 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+  async findAutocompleteLocation(value) {
```

Comment:
@jimboweb you don't need to pass `this.lmap` to `dispatch` if you bind `this` to the `findAutocompleteLocation` class method:

```js
// in body of the LeafletMap class
findAutocompleteLocation =  async (value) => {
  // everything else as is
}
```

Reply:
alternatively in the `constructor` you could do:

```js
this.findAutocompleteLocation = this.findAutocompleteLocation.bind(this)
```

It's one of those dumb javascript things with classes...

Reply:
An arrow function! Yes that solved it, thanks.